package.json

```diff
@@ -53,15 +54,76 @@
     }
   },
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build:esm": "esbuild --bundle --format=esm src/url-pattern.ts --outfile=dist/urlpattern.js",
-    "build:cjs": "esbuild --bundle --format=cjs src/url-pattern.ts --outfile=dist/urlpattern.cjs",
-    "copy:dts": "cp ./src/index.d.ts ./src/types.d.ts ./dist",
-    "build": "npm run build:esm && npm run build:cjs && npm run copy:dts",
-    "pretest": "npm run build; rm -rf node_modules/urlpattern-polyfill; ln -s $(pwd) node_modules/urlpattern-polyfill",
-    "test": "ava --fail-fast -s",
-    "manual-test": "npx http-server -o /index.html -p 4203",
-    "publish-dev": "npm test && VERSION=${npm_package_version%-*}-dev.`git rev-parse --short HEAD` && npm version --no-git-tag-version $VERSION && npm publish --tag dev"
+    "build:esm": "wireit",
+    "build:cjs": "wireit",
+    "build": "wireit",
+    "copyTypeFiles": "wireit",
+    "prepFakeNodeModules": "wireit",
```

Comment:
Do you need this here when you are not going to call it manually from the command line?

Reply:
It seems it's mandatory. I was wondering the same thing.